doc/modules/ROOT/pages/design/overview.adoc

```diff
@@ -8,11 +8,7 @@ and asynchronous (in contrast to most REPLs that build on top of streams
 provided by e.g.  terminals).
 
 TIP: If you are interested in historical context, check the
-https://docs.google.com/document/edit?id=1dnb1ONTpK9ttO5W4thxiXkU5Ki89gK62anRqKEK4YZI&authkey=CMuszuMI&hl=en#[original
-design notes], https://github.com/clojure/tools.nrepl/wiki/nREPL.Next[nrepl
-0.2.0 notes], and
-http://groups.google.com/group/clojure-dev/browse_frm/thread/6e366c1d0eaeec59[discussion]
```

Comment:
I think this link is missing the in the additional resources, so it'd be nice to add it there